CRM/Mailing/BAO/MailingJob.php

```diff
@@ -1042,4 +1041,24 @@ public static function findPendingTasks($jobId, $medium) {
     return $eq;
   }
 
+  /**
+   * Delete the mailing job.
+   *
+   * @param int $id
+   *   Mailing Job id.
+   *
+   * @return mixed
+   */
+  public static function deleteMailingJob($id) {
```

Comment:
I can see on Mailing the function is called 'del' - I think that is kind of the standard  & what the api calls?

Reply:
oops that was my mistake, changed it now :)